bin/prepare_confounders.jl

```diff
@@ -18,6 +18,8 @@ function parse_commandline()
         "--qcfile"
             help = "Path to the UKBiobank ukb_snp_qc.txt"
             arg_type = String
+            default = nothing
```

Comment:
Did you have any problem before? I think those should be the defaults with a `--` option.